src/loaddb/load_server_loader.cpp

```diff
@@ -315,14 +314,15 @@ namespace cubload
       }
 
     const std::vector<std::string> &classes_ignored = m_session.get_args ().ignore_classes;
-    std::string class_name (classname);
     bool is_ignored;
 
-    std::transform (class_name.begin (), class_name.end (), class_name.begin (), [] (unsigned char c)
-    {
-      return std::tolower (c);
-    });
+    char lower_case_string[DB_MAX_CLASS_LENGTH] = { 0 };
+    int str_size = intl_identifier_lower_string_size (classname);
```

Comment:
You may directly call intl_identifier_lower:
```suggestion
    int str_size = intl_identifier_lower (classname, lower_case_string);
    assert (str_size < DB_MAX_CLASS_LENGTH); // or is it <=?
```
Same for the other usage.